### PR TITLE
Additional junk vendor items

### PR DESCRIPTION
**Pull request by Harley-Nation**
Revises green gear to lvl 70 and adds multiple junk items including items from Tymora's Lucky Enchantment.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -5152,9 +5152,21 @@ function addProfile(profession, profile, base){
...
+            _vendorItems[_vendorItems.length] = {
+                pattern: /^Object_Trinket_/,
+                limit: 0
+            }; 
+             _vendorItems[_vendorItems.length] = {
+                pattern: /^Object_Skill_/,
+                limit: 0
+            };
+             _vendorItems[_vendorItems.length] = {
+                pattern: /^Magic Focus$/,
```
Comment (Phr33d0m) [inline main.js]: It's not "Magic Focus". It's Object_Skill_Arcana_Focus and that's already covered in your 2nd _vendorItems pattern added here.

Comment (Harley-Nation) [inline main.js]: The green gear, trinket and magic focus$ came from this thead:

https://greasyfork.org/en/forum/discussion/3861/x

but honestly I haven't seen a Magic Focus$ item.   I have used these additions to my script for about 2 months now with out issue and believe they are of good use.

Comment (Phr33d0m) [inline main.js]: I'm just saying the 'magic focus' thing is useless. Remove it and this is ready for merge.

**Comment by Harley-Nation** [comment]
revised thanks

File: main.js [hunk 2]
```diff
@@ -5152,9 +5152,17 @@ function addProfile(profession, profile, base){
                 limit: 0
             };
             _vendorItems[_vendorItems.length] = {
-                pattern: /_Green_T[1-5]_Unid$/,
+                pattern: /_Green_T[1-7]_Unid$/,
```
Comment (Phr33d0m) [inline main.js:5155]: @noonereally what do you think about this?

Before mod6 we sold up to Green_T5 so people gathering greens (Green_T6) for refining were safe to do so.

I guess we should keep that way and now sell only to Green_T6 so lvl 70 greens can still be kept in the inventory.

Comment (noonereally) [inline main.js:5155]: I don't think anyone that uses the AD income from leadership with the script will try to store unstackable green stuff, that gives less than a single peridot.
But it always can be split into a different option.
1-6 + 7

In my opinion, green stuff should be sold.

~~The only issue I can see is that if someone buys AH gear for an alt or leveling an alt, but I think the vendoring will work for unidentified items only. Not sure though.~~ The regex is for unidentified, didn't notice it.

Perhaps splitting it to 2 options will satisfy everyone.

**Comment by noonereally** [comment]
Thank you for the PR, the vendor stuff could really use an update. Also if someone looks into it, the exclusion filter is outdated, there are higher ranked enchants and other things that I think should be there.

**Pull request merged**